### DISCUSSION TITLE
OnlyOffice in AIO

### DISCUSSION THREAD

**Discussion by andreas-traexler** [nextcloud/all-in-one#1140] [Ideas] [answered]
Hi,
I just wanted to ask, if there should be a second thought given regarding the removal of OnlyOffice from the AIO interface.

I have read the reasoning behind it, though I think that Nextcloud Office is **not** the preferable office for AIO users, as stated in #1031, as it requires stronger server-side hardware for it to even work with medium sized sheets, which makes it not "work for most users" (in my opinion).
Also the expected compatibility with DOCX etc. is not there, also something to consider.

If there really are no plans to rollback its removal, then at least there should be an easy way of manually installing **and maintaining** it (the current ways do not play well with easy updates, certificates etc.) afterwards.

What is the teams opinion on this?

BR

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one/pull/1024/files#diff-a9d0efd05e378456e6ed51949d00a59048f03e2c5f48653291bbe9de700832cfL463-R463
